cmd/observe/flows.go

```diff
@@ -418,6 +418,9 @@ func newFlowsCmdHelper(usage cmdUsage, vp *viper.Viper, ofilter *flowFilter) *co
 	filterFlags.Var(filterVar(
 		"to-identity", ofilter,
 		"Show all flows terminating at an endpoint with the given security identity"))
+	filterFlags.Var(filterVar(
+		"traffic-direction", ofilter,
+		"Show all flows in the given traffic direction (either ingress or egress)"))
```

Comment:
Could we please add completion for the flag's argument given we know it can only be `ingress` or `egress`?